osmaxx/excerptexport/templates/excerptexport/partials/export_detail_string.html

```diff
@@ -1,7 +1,7 @@
 {% load i18n capture_as %}{{ export.get_file_format_display }} /
-{% firstof export.finished_at|date:"Y-m-d H:i:s" export.updated_at|date:"Y-m-d H:i:s" as finished_date %}
```

Comment:
Instead of repeating this format, we could just use it as default setting: https://docs.djangoproject.com/en/1.10/ref/settings/#std:setting-DATETIME_FORMAT
`{{ value|date:"DATETIME_FORMAT" }}` 
Also see https://docs.djangoproject.com/en/1.10/ref/templates/builtins/#date